docs/UserGuide.md

```diff
@@ -246,7 +246,7 @@ Examples:
 
 Switches between different lists in InternConnect.
 
-Format: `checkout file_name`
+Format: `checkout FILE_NAME`
 
 * New and existing lists will be stored at `data/file_name.json`
```

Comment:
FILE_NAME here

Reply:
Line changed to `* New and existing lists will be stored at data/FILE_NAME.json`